robot_modeling/DQ_SerialManipulator.m

```diff
@@ -161,7 +163,7 @@ function set_joint_type(obj, joint_type, ith_joint)
         end
 
         function ret = get_joint_type(obj, ith_joint)
-            % GET_JOINT_TYPE(ith_joint) returns the joint type of the ith joint.
+            % GET_JOINT_TYPE(ith_joint) returns a vector containing the joint types.
```

Comment:
@juanjqo, is `get_joint_type` returning a vector? I thought we were sticking with your initial implementation, in which `get_joint_type` would return just a double.

Reply:
@bvadorno I was a typo. You're right, get_joint_type returns a double.